addons/website_slides/views/slide_slide_views.xml

```diff
@@ -332,9 +332,13 @@
             <field name="arch" type="xml">
                 <search string="Search Contents">
                     <field name="name"/>
+                    <field name="channel_id"/>
+                    <field name="user_id"/>
+                    <field name="tag_ids"/>
```

Comment:
Seems a bit unrelated as well, it could stay in this commit as it's a small change but it should be mentioned in the commit message.

Reply:
I'll update commit message with this info :+1: